core/pom.xml

```diff
@@ -153,8 +143,12 @@
 			<artifactId>opennlp-tools</artifactId>
 		</dependency>
 		<dependency>
+			<!-- TODO: NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH!".
+            This requires a wrapper but the one in https://github.com/bwaldvogel/liblinear-weka is outdated. -->
 			<groupId>de.bwaldvogel</groupId>
 			<artifactId>liblinear</artifactId>
+			<version>2.11</version>
```

Comment:
Actually, different modules in DKPro Core use different versions of liblinear. It is not our choice - the underlying NLP tools require these different versions. liblinear 1.7.1 was created to be compatible with matetools. However, maltparser requires liblinear 1.8. Between 1.7.1 and 1.8, the names of the class package was changed from "liblinear" to "de.bwaldvogel.liblinear". matetools and maltparser cannot be easily combined in the same application.